traits/trait_set_object.py

```diff
@@ -38,88 +37,73 @@ def __init__(self, removed=None, added=None):
 
         if removed is None:
             removed = set()
+        self.removed = removed
+
         if added is None:
             added = set()
-
-        self.removed = removed
         self.added = added
 
 
+# Default item validator for TraitSet.
+
+def accept_anything(item):
+    """
+    Item validator which accepts any item and returns it unaltered.
+    """
+    return item
+
+
 class TraitSet(set):
     """ A subclass of set that validates and notifies listeners of changes.
 
     Parameters
     ----------
-    value : set
-        The value for the set
-    validator : callable
-        Called to validate items in the set
+    value : iterable
+        Iterable providing the items for the set.
+    item_validator : callable, optional
+        Called to validate and/or transform items added to the list. The
+        callable should accept a single item from the set and return
+        the transformed item, raising TraitError for invalid items. If
+        not given, no item validation is performed.
     notifiers : list of callable
         A list of callables with the signature::
 
             notifier(removed, added)
 
+        Where 'added' is a set containing the new values that will be added.
+        And 'removed' is a set containing the old values that will be removed.
```

Comment:
Nit: When the notifier is called, the change has happened. So the future tense may be better changed to either present or present perfect tense? I am also no native speaker to be relied on for such things... :(